controllers/atlasmigration_controller.go

```diff
@@ -166,7 +167,7 @@ func (r *AtlasMigrationReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 	defer wd.Close()
 	// Reconcile given resource
-	status, err := r.reconcile(ctx, wd.Path(), data.EnvName)
+	status, err := r.reconcile(ctx, wd.Path(), data.EnvName, data.BaselineVersion)
```

Comment:
Let's define `baseline` in the `atlas.hcl` template so that you don't need to pass the new argument through functions. You can find the template here. https://github.com/ariga/atlas-operator/blob/701811a3eacf4df3c8382a2db873c3a1ff78cf2e/controllers/templates/atlas_migration.tmpl#L27